src/Uno.UI/UI/Xaml/UIElement.skia.cs

```diff
@@ -1,4 +1,6 @@
-﻿//#define ENABLE_CONTAINER_VISUAL_TRACKING
+﻿#if DEBUG
+#define ENABLE_CONTAINER_VISUAL_TRACKING
+#endif
```

Comment:
Should it be removed?

Reply:
Don't think so. Only few string and much easier for users who tries to understand what is hapening.